api/graphql/schema.graphql

```diff
@@ -659,6 +675,9 @@ input CreateUserInput {
     communityExperiences: CommunityExperienceHasMany
     educationExperiences: EducationExperienceHasMany
     awardExperiences: AwardExperienceHasMany
+
+    expectedClassificationRoles:[ClassificationRole] @rename(attribute: "role_name")
```

Comment:
For this to work, the relationship must also be added to app/Models/Classifications.php
```suggestion
    classificationRoles: [ClassificationRole] @hasMany
```